src/server.ts

```diff
@@ -66,6 +60,34 @@ export class Chat extends AIChatAgent<Env> {
 ${unstable_getSchedulePrompt({ date: new Date() })}
 
 If the user asks to schedule a task, use the schedule tool to schedule the task.
+
+The user can authenticate for PDF upload functionality through the UI. Once authenticated, you can help them with PDF uploads and processing.
+
+**PDF Upload Flow:**
+When a user wants to upload a PDF file, follow this process:
+
+1. **Generate Upload URL**: Use the generatePdfUploadUrl tool to create a presigned upload URL for the file
+2. **User Uploads File**: The user will upload the file directly to R2 storage using the provided URL
+3. **Update Metadata**: After successful upload, use the updatePdfMetadata tool to add description, tags, and file size
+4. **Trigger Ingestion**: Use the ingestPdfFile tool to start processing the uploaded PDF
+
+**Important Session Management:**
```

Comment:
does CF leave cookies in the browser? feature request to stare user session in a cookie (if not stored already). alternatively we can do local storage ig

Reply:
<img width="839" alt="Screenshot 2025-06-25 at 10 00 57 PM" src="https://github.com/user-attachments/assets/f57ece98-ea4d-4f29-814c-c2950ced855a" />